.github/workflows/db.yml

```diff
@@ -0,0 +1,84 @@
+name: Database Build and Test
+
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
```

Comment:
This can just be main can't it?

Reply:
I've simplified that now, but left the `[]` notation in to allow obvious expansion to feature branches